docs/nos-client/api.md

```diff
@@ -228,4 +207,79 @@ nos.send({ asset: GAS, amount, receiver })
     .then((txid) => alert(`${amount} GAS sent in transaction ${txid}`))
     .catch((err) => alert(`Error: ${err.message}`));
 ```
+
+
+## `getPublicKey`
 ---
+The `getPublicKey` provides the public key of the currently authenticated account. It does not require the user to grant permission.
+
+#### Parameters
+None.
+
+#### Returns
+**string** - The public key of the currently signed in user.
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+nos.getPublicKey()
+  .then((publicKey) => alert(`Public Key: ${publicKey}`))
+  .catch((err) => alert(`Error: ${err.message}`));
+```
+
+
+## `encrypt`
+---
+The `encrypt` function allows you to encrypt arbitrary data for another user (you will need his public key, please see `getPublicKey`). It does not require the user to grant permission.
+
+#### Parameters
+* `config` **object** - The config options to perform this operation.
+* `config.recipientPublicKey` **string** - The public key of the recipient account.
+* `config.data` **string | Buffer** - The data to encrypt.
+
+#### Returns
+**object** - object containing encrypted data and values needed for decryption (`iv`, `mac`, `data`)
+
+#### Example
+```javascript
+const nos = window.NOS.V1;
+
+// for the test data below WIF of the sender account is KxDgvEKzgSBPPfuVfw67oPQBSjidEiqTHURKSDL1R7yGaGYAeYnr
+const recipientPublicKey = '031a6c6fbbdf02ca351745fa86b9ba5a9452d785ac4f7fc2b7548ca2a46c4fcf4a';
```

Comment:
Maybe explain how to obtain this or add the possibility to pass in an address as a parameteger in `getPublickKey`, to convert it to a publicKey